CHANGELOG.md

```diff
@@ -31,24 +31,13 @@ This is an example of a brief overview of the _Major_ or _Minor_ version changes
 
 ---
 
+## [11.9] Icons update
+
+Target (changed!), Goal, Advise, Feeds, Devices, Heart, LightBulb, PiggyBank, ShoppingCart, Umbrella, Car, Battery, Bank
```

Comment:
Did you mean to add the other icon names here? I think you just wanted target, goal, and advise.

Reply:
Missed this @sfrieson-cbi ! yeah I messed this up proper. I thankfully caught this and updated it before merging.

Ideally we can get you setup as a proper reviewer, so that I'm not just force merging these things in and you have a chance to look at them to catch mistakes I make 😉